locales/en.yml

```diff
@@ -186,6 +186,8 @@ en:
           Please visit access.redhat.com to download client binaries.
         service_not_enabled: |-
           '%{service}' service is not enabled.
+        kube_not_supportted: |-
+          Client binary installs for kubernetes service is not supported yet.
```

Comment:
"Installation of Kubernetes client library via the install-cli command  is not supported yet"

If not, you need to at least adjust the current sentence. "installs" is plural, "is" singular. Does not match.